frontend/src/concepts/pipelines/content/pipelinesDetails/pipelineRun/runLogs/LogsTab.tsx

```diff
@@ -133,7 +126,7 @@ const LogsTabForPodName: React.FC<{ podName: string }> = ({ podName }) => {
               )}
               <SplitItem>
                 <SimpleDropdownSelect
-                  isDisabled={podContainers.length === 0}
+                  isDisabled={podContainers.length === 0 || podContainers.length === 1}
```

Comment:
Adhering to the [initial mocks](https://www.figma.com/proto/rJjOtl3nxpW6BAcl0MZr4y/Pipeline-log-viewer?page-id=62%3A1895&type=design&node-id=272-2932&viewport=664%2C-1095%2C0.23&t=80ExnogduAcwD3kx-1&scaling=min-zoom&starting-point-node-id=260%3A2386)